vk-video/src/parser.rs

```diff
@@ -32,8 +32,10 @@ pub struct DecodeInformation {
 }
 
 #[derive(Debug, Clone)]
+#[allow(non_snake_case)]
 pub(crate) struct ReferencePictureInfo {
     pub(crate) id: ReferenceId,
+    pub(crate) LongTermPicNum: Option<u64>,
```

Comment:
I assume the spec uses this case but I think we should stick to rust conventions

Reply:
i think not. The big problem with the case is that there are some variables in the spec which are the same and only differ in the case. To be able to tell which one I'm writing about I decided to stick to using exact names from the spec. I know it's ugly, but it's the only way for us to be able to tell which spec variables we're talking about in the code

Reply:
e.g. there's a variable called `frame_num`, which is a syntax element of slice headers. Later the spec defines the variable `FrameNum`, which is calculated in some way and is different from `frame_num`

Reply:
the spec uses both those names, sometimes in the same section